### DISCUSSION TITLE
Which repo for Debian Bullseye/Bookworm installations

### DISCUSSION THREAD

**Discussion by basilstotz** [ltsp/ltsp#803] [General] [answered]
Hi all,

the manual proposes to use Ubuntu ppa as repo for Debian installations. 

Now i saw, that the offical debian main repo ( https://packages.debian.org/bullseye/ltsp ) offers the package too.

Is it a good idea to use the Debian repo (instead the Ubuntu repo)? 

Basil

**Comment by alkisg** [comment, marked as answer]
Hi, read the announcements of [LTSP 22.01](https://github.com/ltsp/ltsp/issues/1#issuecomment-1019081703) and [LTSP 23.01](https://github.com/ltsp/ltsp/issues/1#issuecomment-1397954759).
If you don't care about these fixes/improvements/features, then you can use the bullseye version, 21.01.

If you care about them, add the PPA. It's not an "Ubuntu PPA", it's a repository that is designed to be used by any .deb based distribution, and is tested on Debian.

**Reply by basilstotz** [reply]
Hi alksig,

thank you for your (ultra fast)  answer.  

I checked the 22.01/23.01 announcements and I think I'm fine with 21.01: I'll use the debian repo. 

(And this summer I can even upgrade to 23.01 with bookworm)

Have a nice day

Basil
